src/System.Security.Cryptography.Cng/src/Interop/Interop.Libraries.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+internal static partial class Interop
+{
+    internal static class Libraries
+    {
+        internal const string NCrypt = "ncrypt.dll";
```

Comment:
The CNG interop class is unlikely to be used by any other library (and would require the caller to consume the SSC.Cng contract anyway) so I chose to keep it local. It still follows the conventions so it can be easily merged into common if later appropriate.




In src/System.Security.Cryptography.Cng/src/Interop/Interop.Libraries.cs<https://github.com/dotnet/corefx/pull/2615#discussion_r36243338>:

> @@ -0,0 +1,10 @@

> +// Copyright (c) Microsoft. All rights reserved.

> +// Licensed under the MIT license. See LICENSE file in the project root for full license information.

> +

> +internal static partial class Interop

> +{

> +    internal static class Libraries

> +    {

> +        internal const string NCrypt = "ncrypt.dll";

Why not use src/Common/src/Interop/Windows/Interop.Libraries.cs?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/corefx/pull/2615/files#r36243338>.